autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -289,6 +277,7 @@ def _get_pipeline_steps(
             ("imputer", SimpleImputer(random_state=self.random_state)),
             # ("variance_threshold", VarianceThreshold(random_state=self.random_state)),
             # ("coalescer", CoalescerChoice(default_dataset_properties, random_state=self.random_state)),
+            ("column_splitter", ColumnSplitter(random_state=self.random_state)),
```

Comment:
I think the docstring of the class should be updated to also include `column_splitter` as a step.